tests/dashboard_api_tests.py

```diff
@@ -20,10 +20,11 @@
 from typing import List, Optional
```

Comment:
Would it be useful to move this file to `tests/dashboards/api_tests.py`? Following the implementation structure in the tests could help us find things in the future.